tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/ProgrammeMembershipResource.java

```diff
@@ -61,6 +67,21 @@ public ProgrammeMembershipResource(ProgrammeMembershipService programmeMembershi
     this.programmeMembershipValidator = programmeMembershipValidator;
   }
 
+  /**
+   * This mapper is used to convert the detailed programme membership data into a summary form
+   * by extracting specific fields such as the programme name and start date.
+   */
+  @Mapper
```

Comment:
Is there a reason why this isn't put into the service.mappers package? It feels a bit odd to have it included here.